### PR TITLE
Fix swiper

### PR DESCRIPTION
**Pull request by yuda1124**

### CODE REVIEW

File: shared/components/Create.js [hunk 1]
```diff
@@ -189,7 +190,18 @@ class Create extends Component {
...
       .then((file) =>{
-        this.setState({img: file});
+        const buf = Buffer.from(file, 'base64');
+        imageMagick.resize({
+          srcData: buf,
+          height: 1080,
+          width: 1080
+        }, (err, resized) => {
+          if (err) {
+            // TODO : some error handle code
```
Comment (MechanicKim) [inline shared/components/Create.js]: It might be good to remove useless comment!

Comment (ochanje210) [inline shared/components/Create.js]: todo comments are not useless. it reminds stuffs to be completed.

**Comment by ochanje210** [comment]
shipping this in the top priority due to travis fail problem

**Comment by ochanje210** [comment]
lgtm

**Pull request merged**